src/main/kotlin/com/github/bartimaeusnek/modmixins/mixins/railcraft/boiler/RailcraftBoilerPollution.kt

```diff
@@ -24,12 +25,13 @@ class RailcraftBoilerPollution {
     fun tick(x: Int, c: CallbackInfo) {
         if (this.isBurning)
             tile?.also {
-                if (!it.world.isRemote)
+                if (!it.world.isRemote && (it.worldObj.totalWorldTime % 20).toInt() == 0) {
                     GT_Pollution.addPollution(it.world.getChunkFromBlockCoords(it.x, it.z), when (it) {
-                        is TileMultiBlock -> it.components.size * 2
-                        is TileEngineSteamHobby -> 1
-                        else -> 2
-                })
+                        is TileMultiBlock -> (it.components.size - x) * LoadingConfig.fireboxPollution
```

Comment:
I still dont get why you do `(it.components.size - x)`, where x is the current tick.

Reply:
X is the number of boiler tanks, not the current tick.
So (it.components.size - x) is the number of fireboxes